Jenkinsfile-publish

```diff
@@ -0,0 +1,32 @@
+pipeline {
+  agent none
+
+  parameters {
+    choice(name: "VERSION", choices: ["patch", "minor", "major"], description: "The stage you are building the front end for")
```

Comment:
This is nice. Will we be able to add release notes if we start supporting this properly as an open source library?

Reply:
I don't think `npm version` lets you do it natively but this being npm, there's a package for that.

Reply:
Also [added to the backlog](https://github.com/orgs/nationalarchives/projects/20#card-34784126).